### DISCUSSION TITLE
Proposal to update the Presentation Flow picture with more details

### DISCUSSION THREAD

**Discussion by matgnt** [eclipse-dataspace-dcp/decentralized-claims-protocol#43] [General]
Before I create a PR for this, does it make sense to update the picture for better readability? I always use this separate picture when I want to explain the flow a bit more in detail.

![image](https://github.com/eclipse-tractusx/identity-trust/assets/198707/90e22002-8429-411a-9644-63051bb4208e)

--
Matthias Binzer

**Comment by arnoweiss** [comment]
I think this is quite a good overview. We'd have to strip out all references to other protocols (step 9) and be as generic as possible in the descriptions. For instance, step 4 & 8 require a bit more than currently mentioned.

**Comment by matgnt** [comment]
@arnoweiss feel free to use the flow and adapt it. Here is the source:

```
skinparam boxpadding 30

autonumber

box "Client Entity\n(Consumer)"
participant "Client" as C
participant "Secure Token Service" as STS
participant "Credential Service\n(~Wallet)" as CS
end box

box "Verifier Entity\n(Provider)"
participant "Verifier" as V
end box

C -> STS: SI token request
STS -> C: SI Token response containing an access token

C-> V: Request w/ SI token containing an access token
V -> V: Repackage received\naccess token\ninto own SI token
V-> CS: request w/ Verifier SI token\ncontaining Client access token
CS -> CS: Check Verifier SI token validity\nand contained Client access token
CS ->V: VP response
V -> V: Verify VCs
V -> C: Actual data (e.g. catalog) response

```

**Comment by jimmarino** [comment]
No longer relevant
